vmdb/app/models/configuration_manager_foreman.rb

```diff
@@ -0,0 +1,5 @@
+class ConfigurationManagerForeman < ConfigurationManager
+  delegate :connection_attrs, :to => :provider
+  has_many :configured_systems, :class_name => :ConfiguredSystemForeman
```

Comment:
Will these systems be archived when the config manager is removed?  or should the system be dependent destroyed?